bitrise.yml

```diff
@@ -50,7 +50,7 @@ workflows:
     - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
     - BITRISE_SCHEME: ios-simple-objc
     - SIMULATOR_DEVICE: iPhone 8 Plus
-    - SIMULATOR_OS_VERSION: latest
+    - SIMULATOR_OS_VERSION: "latest"
```

Comment:
what is the reason for this change?

Reply:
This was not intended in itself only is a side result of experimenting with this input